activesupport/test/cache/behaviors/cache_logging_behavior.rb

```diff
@@ -77,7 +86,7 @@ def assert_logs(pattern, &block)
       assert_match pattern, io.string
     end
 
-    def key_pattern(key, namespace: @namespace)
+    def key_pattern(key, namespace:)
```

Comment:
Is this change and all the above changes to accommodate setting `@namespace` in `MemoryStoreTest`?  Or is there some other benefit?

Reply:
I do understand though that testing an un-namespaced memcache or redis store is problematic, so will change this back